docs/root/configuration/overview/_include/xds_api/dynamic-resources.yaml

```diff
@@ -0,0 +1,48 @@
+node:
+  # set <cluster identifier>
```

Comment:
lets remove these comments (and elsewhere)

Reply:
we need the node - id and cluster here